support-frontend/assets/components/subscriptionCheckouts/address/addressFields.jsx

```diff
@@ -2,19 +2,18 @@
 
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { compose } from 'redux';
+import { TextInput } from '@guardian/src-text-input';
+import { css } from '@emotion/core';
+import { space } from '@guardian/src-foundations';
 
 import {
   firstError,
   type FormError,
 } from 'helpers/subscriptionsForms/validation';
 
-import { Input } from 'components/forms/input';
-import { Select } from 'components/forms/select';
+// import { Select } from 'components/forms/select';
```

Comment:
Commented code left in here that I'm assuming can be removed.

Reply:
Yes, thank you!